tests/bootstrap_project/test/util.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:io';
```

Comment:
Discussion/Suggestion: I would say it is fine to add these utils to the lib of this project simply because the business of the project is running these tests.

Reply:
I agree that would be fine, but I have a strong preference toward co-location as it makes it much easier to navigate and quickly understand how the code is used. Is it okay to keep it here for that reason?

Reply:
I agree it makes sense to have the same structure. But in the `cloud` project we have utils in `test/test_util`? Or what test projects logic are you referring to? 🧐

Reply:
As discussed offline I will put this in `lib`, as the entire folder here is just about the test (testing something else).

Reply:
7608016